webclient/src/views/view/view-view.inc

```diff
@@ -327,6 +332,48 @@
       </div>
       <div class="divider" style="margin-top: 10px"></div>
     </div>
+    <!-- Modal-Roomfinder -->
+    <div
+      class="modal modal-lg active"
```

Comment:
```suggestion
      class="modal modal-lg"
```
The reason it's always open is because the active is still given as a class (and it overwrites vue there)